src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -276,9 +276,7 @@ private void InstallButton_Clicked(object sender, RoutedEventArgs e)
 
         private void Versions_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
-            PackageDetailControlModel.PreviousSelectedVersion = e.AddedItems.Count > 0 ? e.AddedItems[0].ToString() : string.Empty;
-
-            return;
+            PackageDetailControlModel.PreviousSelectedVersion = e.AddedItems.Count > 0 ? e.AddedItems[0]?.ToString() : string.Empty;
```

Comment:
This is not related to Transitve Dependencies fix, but it fixes an error for null references on telemetry.